packages/shared/ReactFeatureFlags.js

```diff
@@ -175,3 +175,5 @@ export const enableSyncDefaultUpdates = true;
 export const allowConcurrentByDefault = false;
 
 export const enablePersistentOffscreenHostContainer = false;
+
+export const enableConsoleLogsInDoubleRender = false;
```

Comment:
nit Thinking out loud...I wonder if we could pick a name that also mentions DevTools here?

Like `consoleManagedByDevToolsDuringStrictMode` or something?

I dunno maybe that sucks 😂 I don't feel strongly about this. Feel free to ignore.